app/components/schedule/Fullitem.vue

```diff
@@ -94,61 +91,18 @@ export default {
     }
   },
   computed: {
-    getToday () {
-      const d = new Date()
-      const year = d.getFullYear()
-      const month = (d.getMonth() + 1).toLocaleString('en-US', { minimumIntegerDigits: 2 })
-      const day = d.getDate().toLocaleString('en-US', { minimumIntegerDigits: 2 })
-      return `${year}-${month}-${day}`
-    },
+    ...mapGetters({   
+      streamShowTitle: 'player/getStreamShowTitle',
+      streamEpisodeTitle: 'player/getStreamEpisodeTitle',
+      onAirImage: 'player/getStreamEpisodeImageURL'
+    }),
     isTouchEnabled () {
       return ('ontouchstart' in window) || (navigator.maxTouchPoints > 0) || (navigator.msMaxTouchPoints > 0)
     },
-    streamShowTitle () {
-      if (!this.nowPlaying.now_playing) {
-        return false
-      } else if (this.nowPlaying?.live?.is_live) {
-        return this.nowPlaying?.live?.streamer_name
-      } else {
-        return this.nowPlaying?.now_playing?.song.artist
-      }
-    },
-    streamEpisodeTitle () {
-      if (!this.nowPlaying.now_playing) {
-        return false
-      } else if (this.nowPlaying?.live?.is_live) {
-        return this.nowPlaying?.live?.song?.title || 'Live stream'
-      } else {
-        return this.nowPlaying?.now_playing?.song?.title
-      }
-    },
-    onAirImage () {
-      if (!this.nowPlaying.now_playing) {
-        return false
-      }
-      let streamImage
-      streamImage = this.nowPlaying.now_playing?.song?.art
-      if (this.nowPlaying.live.is_live) {
-        streamImage = this.show.cover_image_url
-      }
-      return this.showAirCheck(this.show.name) ? streamImage : this.show.cover_image_url
-    },
     onAirDescription () {
-      if (!this.nowPlaying.now_playing && this.latestEpisodeData) {
-        return false
-      }
-      if (this.nowPlaying?.live?.is_live) {
-        return this.show.description
-      }
       const descriptionFromArcsi = this.latestEpisodeData?.description
       return descriptionFromArcsi || this.show.description
     },
-    latestEpisodeImage () {
-      if (!this.latestEpisodeData) {
-        return false
-      }
-      return this.show.cover_image_url
-    },
     latestEpisodeTitle () {
```

Comment:
Shouldn't we rename this to latestArchivedEpisodeTitle, and latestEpisodeLink to latestArchivedEpisodeLink on L113?

Reply:
You're right, now that we only request unarchived items. can we handle it as future work? I wouldn't want to risk refactoring now.

Reply:
Yepp, I can create issues/cards for them.